treq/api.py

```diff
@@ -106,9 +109,10 @@ def request(method, url, **kwargs):
 #
 
 def _client(*args, **kwargs):
+    policy = kwargs.get('policy', BrowserLikePolicyForHTTPS())
```

Comment:
the `>=` in `setup.py` and winnowing down the build matrix is really the main thing.

We should probably do a release before landing that drop in support though.